src/Acuminator/Acuminator.Utilities/DiagnosticSuppression/SuppressionManager.cs

```diff
@@ -72,6 +54,19 @@ public void ReloadFile(object sender, SuppressionFileEventArgs e)
 			_fileByAssembly[assembly] = newFile;
 		}
 
+		private (SuppressionFile File, string Assembly) LoadFileAndTrackItsChanges(string suppressionFilePath, bool generateSuppressionBase)
+		{
+			lock (_fileSystemService)
```

Comment:
@SENya1990 why do we need a look here?

Reply:
@metallmonster because the VS throws exception about access to file from another thread.